tests/e2e/onpremises/output.tf

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+output "haproxy_ip" {
+  value = hcloud_server.haproxy.ipv4_address
+}
+
+output "ingress_domain" {
+  value = "ingress.${replace(hcloud_server.haproxy.ipv4_address, ".", "-")}.nip.io"
+}
+
+output "furyctl_yaml" {
+  value = <<EOF
+---
+apiVersion: kfd.sighup.io/v1alpha2
+kind: OnPremises
+metadata:
+  name: reevo
+spec:
+  distributionVersion: v1.31.1
+  kubernetes:
+    pkiFolder: ./pki
+    ssh:
+      username: root
+      keyPath: /cache/ci-ssh-key
+    dnsZone: nip.io
+    controlPlaneAddress: ${replace(hcloud_server.haproxy.ipv4_address, ".", "-")}.nip.io:6443
+    podCidr: 10.128.0.0/14
+    svcCidr: 172.30.0.0/16
+    loadBalancers:
+      enabled: true
+      hosts:
+        - name: haproxy1-10-10-1-2
+          ip: 10.10.1.2
+      keepalived:
+        enabled: false
+        interface: enp0s6
+        ip: 10.250.250.179/32
+        virtualRouterId: "201"
+        passphrase: "b16cf069"
+      stats:
+        username: admin
+        password: "password"
+      additionalConfig: "{file://./haproxy-additional.cfg}"
+    masters:
+      hosts:
+        - name: master0-10-10-1-3
+          ip: 10.10.1.3
+        - name: master1-10-10-1-4
+          ip: 10.10.1.4
+        - name: master2-10-10-1-5
+          ip: 10.10.1.5
+    nodes:
+      - name: infra
+        hosts:
+          - name: infra0-10-10-1-6
+            ip: 10.10.1.6
+          - name: infra1-10-10-1-7
+            ip: 10.10.1.7
+          - name: infra2-10-10-1-8
+            ip: 10.10.1.8
+        taints: []
+      - name: worker
+        hosts:
+          - name: worker0-10-10-1-9
+            ip: 10.10.1.9
+          - name: worker1-10-10-1-10
+            ip: 10.10.1.10
+          #- name: worker2-10-10-1-11
+          #  ip: 10.10.1.11
+          #- name: worker3-10-10-1-12
+          #  ip: 10.10.1.12
+          #- name: worker4-10-10-1-13
+          #  ip: 10.10.1.13
+        taints: []
+    #advancedAnsible:
+      #config: |
+      #  ansible_ssh_common_args='-o ProxyCommand="ssh -W %h:%p -q root@${hcloud_server.haproxy.ipv4_address}"'
```

Comment:
let's delete this if not in use
```suggestion
```

Reply:
I left it there for future reference, maybe we are able to make it work (i was not)